docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+
+services:
+  jekyll:
+    user: "${UID}:${GID}"
+    build: .
+    command: sh -c "chown $UID / && echo NODE && node -v && bundle exec jekyll serve --incremental --host=0.0.0.0 "
```

Comment:
Do we need `--watch` here (like the README.md says)?

Reply:
That would be more consistent, indeed.